src/main/java/com/epam/lab/paymentsystem/utility/convector/Convector.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.lab.paymentsystem.utility.convector;
+
+import com.epam.lab.paymentsystem.entities.User;
+
+public final class Convector {
```

Comment:
'Transoformer' or 'Convertor' is a better name. Are you planning to use this class only for Users?

Reply:
Also what is a purpose of this class and 'convertUser()' method if we already have 'getCopy(User old)'?

Reply:
Ok, I see that you tried to move this method out of UserDAOImpl. But the first question is still relevant